torch/_dynamo/variables/builtin.py

```diff
@@ -952,6 +952,16 @@ def call_method(
         args: "List[VariableTracker]",
         kwargs: "Dict[str, VariableTracker]",
     ) -> "VariableTracker":
+        if self.fn == object and name == "__setattr__":
+            assert len(args) == 3
+            assert len(kwargs) == 0
```

Comment:
Although rare in practice, can't these be listed as kwargs too `self=, name=, value=`? We should handle that edge case too

Reply:
`object.__setattr__` takes no keyword arguments:

```python
In [1]: class A:
   ...:     pass
   ...:     

In [2]: a = A()

In [3]: object.__setattr__(a, 'x', 1)  # success

In [4]: object.__setattr__(a, name='x', value=1)
TypeError: wrapper __setattr__() takes no keyword arguments
```

Reply:
oh interesting, I didn't realize it's position only.